src/mixins/dropdown.js

```diff
@@ -347,12 +347,10 @@ export default {
       // Note: Inputs are in a special .dropdown-form container
     },
     onFocusOut (evt) {
-      if (this.$el.contains(evt.relatedTarget)) {
+      if (this.$el.contains(evt.target)) {
```

Comment:
`evt.relatedTarget` is the correct value to check, as it refers to the element that is _gaining_ focus, not the element that is _loosing_ focus (for a `focusout` event).

We want to see if the element that just received the focus outside of the dropdown menu (as we are listening at the root of the dropdown), and using `evt.target` makes it so that the dropdown never closes on clickout or tab-out.

Perhaps, it might be better for us to instantiate a `focusout` listener, whenever the dropdown is open, on the document element, and then we could test for `evt.target` being inside the dropdown.

Reply:
@tmorehouse Thanks for pointing out. Do you have a concrete solution for this in mind or should I try to figure it out over the weekend?

Nevertheless, my previous solution isn't working reliable and we need to come up with a better solution.

Reply:
You can take a peek at the latest `b-modal` code's `enforceFocus` handler, which is now adding/removing a listener on the `document` whenever the modal is open.

You could basically enable the listener when the dropdown is `shown`, and remove it on `hidden` (or un-cancelled `hide`, and also on `beforeDestroy()`).  It would be a few extra lines of code, but I think it may address the issue.

Reply:
Thanks for the hint. Will look into it over the weekend 👍

Reply:
You could add the `focusout` (or possibly `focusin` instead) registration via `addEvent()`/`removeEvent(...)` to the `whileOpenListen` method (https://github.com/bootstrap-vue/bootstrap-vue/blob/dev/src/mixins/dropdown.js#L232) method in the dropdown mixin, and remove it off of the dropdown container.

Reply:
Hmmm.. Safari Mobile or desktop?

Reply:
Desktop

Reply:
OK.. I know mobile has issues with certain things